src/assets/i18n/fr.json5

```diff
@@ -313,6 +313,33 @@
   // "admin.registries.schema.title": "Metadata Schema Registry",
   "admin.registries.schema.title": "Registre des schémas de métadonnées",
 
+    //"admin.access-control.bulk-access.breadcrumbs": "Bulk Access Management",
+   "admin.access-control.bulk-access.breadcrumbs": "Gestion des accès en lot",
+   
+  //"administrativeBulkAccess.search.results.head": "Search Results",
+  "administrativeBulkAccess.search.results.head": "Résultats de recherche",
+  
+  //"admin.access-control.bulk-access": "Bulk Access Management",
+  "admin.access-control.bulk-access": "Gestion des accès en lot",
+
+  //"admin.access-control.bulk-access.title": "Bulk Access Management",
+  "admin.access-control.bulk-access.title": "Gestion des accès en lot"
+  
+
+  //"admin.access-control.bulk-access-browse.header": "Step 1: Select Objects",
+  "admin.access-control.bulk-access-browse.header": "Étape 1 : Sélectionner les objets",
+
+  //"admin.access-control.bulk-access-browse.search.header": "Search",
+  "admin.access-control.bulk-access-browse.search.header": "Chercher",
+
+
+  //"admin.access-control.bulk-access-browse.selected.header": "Current selection({{number}})",
+  "admin.access-control.bulk-access-browse.selected.header": "Sélection actuelle({{number}})",
+
+
+  //"admin.access-control.bulk-access-settings.header": "Step 2: Operation to Perform",
+  "admin.access-control.bulk-access-settings.header": "Étape 2 : Opération à éxecuter",
```

Comment:
Should be "Étape 2 : Opération à exécuter"

Reply:
Done in last commit. 46df7e805e2eac71c31acd35bd15367ceb5efc46